src/CountryRepository.php

```diff
@@ -2727,6 +2758,17 @@ class CountryRepository
             'currencyIsoAlphaCode' => 'MAD',
             'isoAlpha2' => 'EH',
         ],
+        'XK' => [
```

Comment:
I am not sure we should add it. Kosovo does not have assigned ISO 3166 code, only user assigned used by several organizations:

https://en.wikipedia.org/wiki/XK_(user_assigned_code)

> ISO affirms that no [code beginning with "X"](https://en.wikipedia.org/wiki/ISO_3166#Codes_beginning_with_%22X%22) will ever be standardised as a country code. ("XK" for [Kosovo](https://en.wikipedia.org/wiki/Kosovo) is a unilateral "user assigned code" and not an ISO 3166 standard country code.) Consequently if/when Kosovo is recognised, this XK code will need to be replaced with one beginning with another letter.